doc/userguide/rules/payload-keywords.rst

```diff
@@ -290,6 +306,10 @@ If one or more ``content`` keywords precedes ``bsize``, each occurrence of ``con
 will be inspected and an error will be raised if the content length and the bsize
 value prevent a match.
 
+bsize will not match if a sticky buffer is absent.
+``bsize: 0``  means a buffer exists and is empty (like a empty HTTP header value).
+You should use the specific keyword ``absent`` to test if a buffer is absent.
+
```

Comment:
I think this could become a `.. note::` to have more emphasis.
nit: _an_ empty HTTP header value

Reply:
Thanks Juliana, will fix after merge of first #10462